### PR TITLE
v1.2.0

### PR DESCRIPTION
**Pull request by Zoey2936**
- allow to throw players #63 - can be done by hitting stacked player - fall damage of a thrown player can be enabled/disabled
- lint repository
- stack mode is now default on
- fix language file (immmune => immune)
- update to 1.20.4

### CODE REVIEW

File: src/main/java/de/zoeyvid/stacker/StackerCommand.java [hunk 1]
```diff
@@ -8,144 +10,140 @@
...
-                } else {
-                    main.getStackmode().add(player);
-                    player.sendMessage(main.getPrefix() + ChatColor.WHITE + loadLanguage.joinStackmode());
-                    return true;
-                }
+        return true;
+      } else if (args[0].equalsIgnoreCase("help")) {
+        if (player.hasPermission("stacker.help")) {
+          player.sendMessage(main.getPrefix() + ChatColor.WHITE + loadLanguage.helpTitle());
+          player.sendMessage(main.getPrefix() + ChatColor.YELLOW + "/stacker " + ChatColor.WHITE + loadLanguage.helpNormal());
```
Comment (Zoey2936) [inline src/main/java/de/zoeyvid/stacker/StackerCommand.java]:
```suggestion
          player.sendMessage(main.getPrefix() + ChatColor.YELLOW + "/stacker " + ChatColor.WHITE + loadLanguage.helpImmune());
```

File: src/main/resources/plugin.yml
Comment (DavidCraftDev) [inline src/main/resources/plugin.yml:1]: Add your name to the authors section

File: src/main/java/de/zoeyvid/stacker/main.java [hunk 2]
```diff
@@ -1,149 +1,160 @@
...
 
 public final class main extends JavaPlugin {
 
-    private static final String prefix = ChatColor.GRAY + "[" + ChatColor.AQUA + "Sta" + ChatColor.DARK_AQUA + "cker" + ChatColor.GRAY + "] ";
-    private static final ArrayList stackmode = new ArrayList<Player>();
-    private static final ArrayList disabled = new ArrayList<Player>();
-    private final int langVersion = 2;
-    private FileConfiguration cfg;
-    private FileConfiguration lang;
+  private static final String prefix = ChatColor.GRAY + "[" + ChatColor.AQUA + "Stacker" + ChatColor.GRAY + "] ";
```
Comment (DavidCraftDev) [inline src/main/java/de/zoeyvid/stacker/main.java:19]: Why only one color? But okay xD

File: src/main/java/de/zoeyvid/stacker/loadConfig.java
Comment (DavidCraftDev) [inline src/main/java/de/zoeyvid/stacker/loadConfig.java:1]: Why is the config accessed again after setting the default value? Isn't that a waste of server resources?

File: src/main/resources/config.yml [hunk 3]
```diff
@@ -18,16 +18,22 @@ sendMessage: true
 saveState: true
 #
 # Should a Player be on join default immune?
-defaultImmune: false
+defaultImmune: true
```
Comment (DavidCraftDev) [inline src/main/resources/config.yml]: Why should the players default immune at the default config?

File: src/main/java/de/zoeyvid/stacker/StackerCommand.java
Comment (DavidCraftDev) [inline src/main/java/de/zoeyvid/stacker/StackerCommand.java:1]: Why should the default action of the command be to make himself immune?

File: src/main/resources/config.yml (hunk 3 above)
Comment (Zoey2936) [reply src/main/resources/config.yml]:
```suggestion
defaultImmune: false
```

File: src/main/java/de/zoeyvid/stacker/loadConfig.java [hunk 4]
```diff
@@ -1,114 +1,134 @@
...
-        } else {
-            cfg.set("defaultImmune", false);
-            saveConfig();
-            return false;
-        }
+  public static boolean defaultImmune() {
+    if (cfg.isBoolean("defaultImmune")) {
+      return cfg.getBoolean("defaultImmune");
+    } else {
+      cfg.set("defaultImmune", true);
```
Comment (Zoey2936) [inline src/main/java/de/zoeyvid/stacker/loadConfig.java]:
```suggestion
      cfg.set("defaultImmune", false);
```

**Review by DavidCraftDev — APPROVED** [review]

**Pull request merged**